Added Made with PlayCanvas page with showcase video (#304)
playcanvas/developer.playcanvas.com: content/en/user-manual/made-with-playcanvas.md

---
title: Made with PlayCanvas
template: usermanual-page.tmpl.html
position: 1.6
---

> Below are our showcase videos of some of the best WebGL browser games and experiences made with PlayCanvas.

Reviewer: ```suggestion
Below is our showcase video of some of the best WebGL browser games and experiences made with PlayCanvas.
```

> Find more fantastic games and experiences made by our users at [Awesome PlayCanvas GitHub repo][awesome-playcanvas]!

Reviewer: ```suggestion
Find more fantastic games and experiences made by our users in the [PlayCanvas Awesome List][awesome-playcanvas]!
```
